backend/contextProperties.js

```diff
@@ -1,32 +1,21 @@
 /**
  * Wrapper functions to set commonly used fields on an 'context' object that is managed over the scope of a request.
  * Hopefully reduces the liklihood of mis-typing property names.
- * Note that by convention the controller(s) and Middleware use the res.locals property as the request scoped context.
+ * Note that by convention the controller(s) and Middleware use the req.session property as the request scoped context.
  * From controllers down to clients, client interceptors etc the context object is called 'context'.
  */
 
 /* eslint no-param-reassign: 2 */
 const setTokens = (context, accessToken, refreshToken) => {
-  context.accessToken = accessToken
-  context.refreshToken = refreshToken
+  context.access_token = accessToken
+  context.refresh_token = refreshToken
 }
 
-const hasTokens = context => {
-  if (!context) return false
-  return !(!context.accessToken || !context.refreshToken)
-}
+const hasTokens = context => !!(context && context.access_token && context.refresh_token)
```

Comment:
Might be more readable by replacing the double bang with Boolean(context && context.access_token && context.refresh_token)

Reply:
Like the idea, will do